jlm/tooling/Command.hpp

```diff
@@ -627,15 +631,17 @@ class JlmHlsCommand final : public Command
   Create(
       CommandGraph & commandGraph,
       const util::FilePath & inputFile,
-      const util::FilePath & outputFolder)
+      const util::FilePath & outputFolder,
+      const std::vector<std::string> & options)
   {
-    std::unique_ptr<JlmHlsCommand> command(new JlmHlsCommand(inputFile, outputFolder));
+    auto command = std::make_unique<JlmHlsCommand>(JlmHlsCommand(inputFile, outputFolder, options));
```

Comment:
This can be changed to: `auto command = std::make_unique<JlmHlsCommand>(inputFile, outputFolder, options))`

Reply:
Changed